src/libpanic_unwind/lib.rs

```diff
@@ -69,6 +69,7 @@ mod imp;
 
 // i686-pc-windows-gnu and all others
 #[cfg(any(all(unix, not(target_os = "emscripten")),
+          target_os = "redox",
```

Comment:
Redox could potentially use `libunwind` for backtraces. Right now, panic_abort is preferred.